index.html

```diff
@@ -457,6 +457,527 @@ <h1>Authentication and Access Control</h1>
+          <dd>
+
+            Michael McCool, Cristiano Aguzzi
+
+          </dd>
+          <!--dt>Reviewer(s)</dt>
+          <dd>
+
+
+          </dd>
+          <dt>Tracker Issue ID</dt>
+          <dd>
+
+
+          </dd>
+          <dt>Category</dt>
+          <dd>
+
+
+          </dd>
+          <dt>Class</dt>
+          <dd>
+
+
+          </dd>
+          <dt>Status</dt>
+          <dd>
+
+            WIP
+
+          </dd-->
+          <dt>Target Users</dt>
+          <dd>
+
+            <ul>
+              <li>device owner</li>
+              <li>device user</li>
+              <li>device application</li>
+              <li>service provider</li>
+              <li>identity provider</li>
+              <li>directory service</li>
+            </ul>
+
+          </dd>
+          <dt>Motivation</dt>
+          <dd>
+
+            OAuth 2.0 is an authorization protocol widely known for its usage across several web services.
+            It enables third-party applications to obtain limited access to HTTP services on behalf of the resource
+            owner
+            or of itself.
+            The protocol defines the following actors:
+
+            <ul>
+              <li>Client: an application that wants to use a resource owned by the resource owner. </li>
+              <li>Authorization Server: An intermediary that authorizes the client for a particular `scope`. </li>
+              <li>Resource: a web resource </li>
+              <li>Resource Server: the server where the resource is stored</li>
+              <li>Resource Owner: the owner of a particular web resource. If it is a human is usually referred to as an
+                end-user. More specifically from the RFC:</li>
+              <ul>
+                <li>An entity capable of granting access to a protected resource.</li>
+              </ul>
+            </ul>
+
+            These actors can be mapped to WoT entities:
+            <ul>
+              <li>Client is a WoT Consumer</li>
+              <li>Authorization Server is a third-party service</li>
+              <li>Resource is an interaction affordance</li>
+              <li>Resource Server is a Thing described by a Thing Description acting as a server. </li>
+              May be a device or a service.
+              <li>Resource Owner might be different in each use case.
+                A Thing Description may also combine resources from different owners or web server.</li>
+            </ul>
+            TO DO: Check the OAuth 2.0 spec to determine exactly how Resource Owner is defined.
+            Is it the actual owner of the resource (eg running the web server) or simply someone
+            with the rights to access that resource?
+            <br>
+            The OAuth 2.0 protocol specifies an authorization layer that separates the client from the resource owner.
+            The basic steps of this protocol are summarized in the following diagram:
+            <pre>

```

Comment:
can leave these for now, eventually can do SVG (can be done in cleanup, not necessary right now)